packages/common/src/channelAddress.test.ts

```diff
@@ -0,0 +1,29 @@
+import { generateChannelId, getChannelNameFormChannelId } from './channelAddress'
+
+describe('Generate Channel Id', () => {
+  it('name "rockets" is the channel name', () => {
+    expect(generateChannelId('rockets')).toContain('rockets')
+  })
+
+  it('Should include hexadecimals characters in a determined structure (name + _ + 16 hex)', () => {
+    const channelName = 'rockets'
+    const randomBytesLength = 32 // 16 chars in hex
+    const underscoreLength = 1
+    const expectedLength = channelName.length + underscoreLength + randomBytesLength
+    expect(generateChannelId('rockets')).toHaveLength(expectedLength)
+  })
+})
+
+describe('Get Channel Name From Channel Id', () => {
+  it('Should return the channel name', () => {
+    const channelId = 'rockets_1faff74afc8daff3256275ce89d30528'
+    const channelName = 'rockets'
+    expect(getChannelNameFormChannelId(channelId)).toEqual(channelName)
+  })
+  it('Should return the channel id fi does not match the structure', () => {
```

Comment:
Thank you for adding tests :)
 
There is a typo "fi" -> "if". Also I believe that the function should be named `getChannelNameFromChannelId` ("Form" -> "From"). I'd appreciate if you fixed that too.

Reply:
Yeah, there is a typo in the function name. I fixed it in https://github.com/TryQuiet/quiet/pull/1727. Also, I fixed the typo that I introduced with the `fi`. If you want, we can close this PR and focus on the other one that includes the commits from this PR too.